### ISSUE TITLE
RFE: add support for the parisc arch/ABI

### ISSUE THREAD

**Issue by pcmoore** [seccomp/libseccomp#28]
Initial patches at the link below:

* https://groups.google.com/forum/#!topic/libseccomp/6O28XC3urtY

**Comment by pcmoore** [comment]
As of 4.5-rc4, the upstream kernel does not have HAVE_ARCH_SECCOMP_FILTER enabled for parisc.

**Comment by pcmoore** [comment]
The latest patch: https://groups.google.com/forum/#!msg/libseccomp/6O28XC3urtY/pRmy29tuEgAJ

**Comment by lucab** [comment]
Further patch revision [v3](https://groups.google.com/d/msg/libseccomp/6O28XC3urtY/6kUQpNBgFAAJ):
```
- Stipped out patch which reports if a check was skipped because valgrind isn't installed.
- Added tuxcall pseudo syscall for 19-sim-missing_syscalls testcase
- Added sysmips pseudo syscall for 29-sim-pseudo_syscall testcase 
```

Any plan to finally merge this or does it need further refinements? I've got [a request](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=820501) to incorporate this in Debian package, but I would prefer to see it first accepted here and then just cherry-pick it.

**Comment by pcmoore** [comment]
Quick update: https://groups.google.com/forum/#!topic/libseccomp/6O28XC3urtY

**Comment by pcmoore** [comment]
Resolved by c86e1f565537b28b73ebd63f0239b4a446925534.

**Comment by lucab** [comment]
Great, thanks. I cherry-picked this for Debian, and it is now properly [building+testing](https://buildd.debian.org/status/fetch.php?pkg=libseccomp&arch=hppa&ver=2.3.1-2&stamp=1464467425) on hppa (parisc).
